doc/MAGIC.md

```diff
@@ -737,20 +748,20 @@ The following is a list of possible `values`:
 | `ITEM_DAMAGE_STAB` | 
 
 
-| Melee-only enchantment values  | Description
+| Enchanted item value | Description
 |---                             |---
 | `ITEM_ARMOR_ACID` | 
 | `ITEM_ARMOR_BASH` | 
 | `ITEM_ARMOR_BIO` | 
+| `ITEM_ARMOR_BULLET` | 
 | `ITEM_ARMOR_COLD` | 
 | `ITEM_ARMOR_CUT` | 
 | `ITEM_ARMOR_ELEC` | 
 | `ITEM_ARMOR_HEAT` | 
-| `ITEM_ARMOR_PURE` | 
```

Comment:
Thought this was added recently?

Reply:
I checked a magic_enchantment.h for this, and yeah, is has lack of all of this stuff. For PURE its even make no sense to have an armor, because it designed to penetrate all armor